packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -90,31 +155,41 @@ export class SceneTree {
   @State()
   private scrollTop = 0;
 
-  private computedItemHeight: number | undefined;
-
-  private leftTemplate: HTMLTemplateElement | undefined;
-  private rightTemplate: HTMLTemplateElement | undefined;
-  private bindings = new Map<Element, CollectionBinding>();
-
-  private idleCallbackId: number | undefined;
-
-  private connected = false;
+  @State()
+  private computedRowHeight: number | undefined;
 
-  private onStateChangeDisposable: Disposable | undefined;
-  private subscribeDisposable: Disposable | undefined;
+  /**
+   * This stores state that shouldn't trigger a refresh if the data changes.
+   * Marking this with @State to allow to preserve state across live-reloads.
+   */
+  @State()
+  private stateMap: StateMap = {
+    bindings: new Map(),
+    connected: false,
+  };
 
+  /* eslint-disable lines-between-class-members */
```

Comment:
I disabled the lint rule, because there's value in visually grouping these getters/setters together. The getter and setter are coupled and I don't want these code blocks to get separated as the code evolves. The vast majority of class members should have white space between them, so I think we should keep this enforced.